src/Router.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import App from './components/App';
+import AdminPage from './components/AdminPage';
+
+export default function Router() {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Redirect exact from="/" to="/app" />
```

Comment:
for clarification, this would work like "hostname:port/" => "hostname:port/app" ?

Reply:
Yeah. We can change paths later. We can just have the `/` go to app and not redirect.